### DISCUSSION TITLE
"Update thumbnails" does not work

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#207] [Fixed Issues]
**rAudio 1 • 20210221**

"Update thumbnails"  generates:
```
Addons Progress

/srv/http/bash/albumthumbnail.sh "NAS/Musik/Neue Musik/Shankar, Ravi" 

 .  Update Album Thumbnails ...

/srv/http/bash/albumthumbnail.sh: line 26: NAS/Musik/Neue Musik/Shankar, Ravi
NAS/Musik/Neue Musik/Shankar, Ravi/Symphony: No such file or directory

Duration 00:00

 .  Done
```
RPI 3B + HifiBerry Digi+ Pro + rAudio-1 + aarch64 (experimental) built with rOS

EDIT: After reverting the last commit in /srv/http/bash/albumthumbnail.sh "Update thumbnails" works again:
![Commit](https://user-images.githubusercontent.com/25383513/108632908-9089ab80-7471-11eb-8638-ee61959d882d.jpg)

**Comment by klassikmann** [comment]
I just installed the newly released beta of rAudio 1 64bit as an alternative to the rOS based aarch64 mentioned above.
/srv/http/bash/albumthumbnail.sh didn't change so it was no surprise that the issue ("Update thumbnails" does not work) is still there. Changing line 26 resolves the problem.

**Reply by rern** [reply]
Thank you for pointing out. It's fixed.
